### PR TITLE
Update insertionsort.py code

### PR DESCRIPTION
**Pull request by satishjasthi**
Update the code to perform insertion sort more efficiently compared to previous version of code. This PR is a part of Hacktoberfest.

**Fixes issue:** 
Improve performance of insertionsort.py code

**Changes:**
Entire code is rewritten to perform insertion sort in an efficient way. I have provided time profiling information to compare between older and new version of insertionsort.py code in [modified_insertion_sort_cProfiling_data.txt](https://github.com/OpenGenus/cosmos/files/1404104/modified_insertion_sort_cProfiling_data.txt)

### CODE REVIEW

**Comment by arnavb** [comment]
Please read the [Python Style Guide](https://github.com/OpenGenus/cosmos/blob/master/guides/coding_style/python/style.md) before contributing to this repository!

File: code/sorting/insertion_sort/insertion_sort.py [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
+	print ("Given array : \n",L)
+	print ("Sorted array : \n",insertion_sort(L))
+	
+	
+
+
+
+
+
+
```
Comment (arnavb) [inline code/sorting/insertion_sort/insertion_sort.py]: Get rid of all these spaces.

Comment (satishjasthi) [reply code/sorting/insertion_sort/insertion_sort.py]: Sure

**Comment by satishjasthi** [comment]
Can you merge this update to master? I think it is following all the guidelines. If not please let me know I'll update the code.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by satishjasthi** [comment]
Yes I am

**Pull request merged**